package.json

```diff
@@ -28,11 +28,7 @@
     "test:ci": "run-s test:ci:*",
     "test:dev:ava": "ava",
     "test:ci:ava": "nyc -r lcovonly -r text -r json ava",
-    "prepublishOnly": "run-s prepublishOnly:*",
-    "prepublishOnly:checkout": "git checkout main",
```

Comment:
We don't need those when publishing from CI.
Also I don't think they would work since the checkout action checks out a specific commit.